### ISSUE TITLE
Add plate boundaries as a built-in dataset

### ISSUE THREAD

**Issue by michaelgrund** [GenericMappingTools/gmtserver-admin#100]
**Description of the desired feature**

Since it's one of the most frequently used geo datasets to plot as well as based on a comment in the forum (https://forum.generic-mapping-tools.org/t/borders-plate-tectonics/1946) I think it would be useful to have the plate boundaries of [Bird (2002)](https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2001GC000252) as a built-in dataset in GMT/PyGMT. 

**Are you willing to help implement and maintain this feature?** Yes (but first discuss)

**Comment by PaulWessel** [comment]
I should try to get input from Christian Heine on this.  I believe he did all the hard work of adding -Sf information to the segment headers so that trenches has saw tooth fronts and FZ has strike slip, etc, and that the lines are organized so that overriding plate is always to the left as you move along a line.  The Bird dataset as is requires lots of work.  But otherwise agree.

**Comment by joa-quim** [comment]
I don't think the Bird's plate borders file should be endorsed by GMT. It's very complex and questionable in many places.

**Comment by seisman** [comment]
I think the GMT "cache" files should be only used in GMT/PyGMT/GMT.jl examples and tests.

**Comment by PaulWessel** [comment]
Yes, as the name cache implies, I do not think we have any long-term obligation to maintain anything that is put there. No doubt such a data set is useful to many of us, but once we cross that line of also serving up specific discipline data sets we will be toast.

**Comment by seisman** [comment]
> Yes, as the name cache implies, I do not think we have any long-term obligation to maintain anything that is put there. No doubt such a data set is useful to many of us, but once we cross that line of also serving up specific discipline data sets we will be toast.

I agree. Can we (or do we want to) set up a repository to store "user-contributed dataset"? Users can submit the dataset that can be directly used in GMT (e.g., Bird's plate borders), and **users** are responsible to maintain the datasets.

The [GMT China Community](https://github.com/gmt-china/) is maintaining such a repository [china-geospatial-data](https://github.com/gmt-china/china-geospatial-data), which stores some dataset that are specific for Chinese users.

**Comment by PaulWessel** [comment]
Sure, we could enable such repositories. There would still be decisions (what should go into it - anything? or a theme-based limitation).

**Comment by chhei** [comment]
@michaelgrund, @PaulWessel - The plate boundary data from Bird converted for GMT >5 is available [through my website](https://wiki.paleoearthlabs.org/scientificdata/platetectonics/birdplateboundaries). I'd be more than happy to contribute the file to a community repository.

**Comment by michaelgrund** [comment]
> 
> 
> > Yes, as the name cache implies, I do not think we have any long-term obligation to maintain anything that is put there. No doubt such a data set is useful to many of us, but once we cross that line of also serving up specific discipline data sets we will be toast.
> 
> I agree. Can we (or do we want to) set up a repository to store "user-contributed dataset"? Users can submit the dataset that can be directly used in GMT (e.g., Bird's plate borders), and **users** are responsible to maintain the datasets.
> 
Sounds good to me but I think we need a more detailed discussion about what to add and what not (as @PaulWessel mentioned).

**Comment by Esteban82** [comment]
About the issue of adding new specific datasets (e.g. geological plates), maybe we could make a page (like a page in forum?) where the users can post links to datasets ready to use with GMT. In this way we facilitate the use of them I we don't have to worry about maintain it.
